app/views/state_file/questions/eligible/_nj_credits_unsupported.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="reveal">
+  <p><button class="reveal__button"><%= t("state_file.questions.eligible.edit.credits_not_supported")%></button></p>
+
+  <div class="reveal__content">
+    <div class="spacing-below-25">
+      <ul class="list--bulleted">
+        <%=t('.unsupported_html') %>
+      </ul>
+    </div>
+
+    <div class="spacing-below-25">
+      <b><%=t('.also_unsupported_title') %></b>
```

Comment:
[pebble] I can't completely tell from this markup, but are we using a style to convey semantic meaning to sighted users? Should this be a header?